src/components/VisibilityCard/VisibilityCard.tsx

```diff
@@ -130,53 +137,80 @@ export const VisibilityCard: React.FC<VisibilityCardProps> = props => {
     <Card>
       <CardTitle title={intl.formatMessage(visibilityCardMessages.title)} />
       <CardContent>
-        <RadioSwitchField
+        <RadioGroup
           disabled={disabled}
-          error={!!getFieldError(errors, "isPublished")}
-          firstOptionLabel={
-            <>
-              <p className={classes.label}>{messages.visibleLabel}</p>
-              {isPublished && publicationDate && Date.parse(publicationDate) < dateNow && (
-                <span className={classes.secondLabel}>
-                  {messages.visibleSecondLabel || visibleMessage(publicationDate)}
-                </span>
+          name="isPublished"
+          value={String(isPublished)}
+          onValueChange={value => {
+            onChange({
+              target: {
+                name: "publishedAt",
+                value: value === "false" ? null : availableForPurchaseAt,
+              },
+            });
+            onChange({
+              target: {
+                name: "isPublished",
+                value: value === "true",
+              },
+            });
+          }}
+          display="flex"
+          flexDirection="column"
+          gap={3}
+        >
+          <RadioGroup.Item id={`isPublished-true`} value="true">
+            <Box display="flex" __alignItems="baseline" gap={2}>
+              <Text>{messages.visibleLabel}</Text>
+              {isPublished && publishedAt && Date.parse(publishedAt) < dateNow && (
```

Comment:
this condition is a little difficult to read, i'd extract it but it's a nitpick really - up to you